apps/google-analytics-4/src/components/FormControlServiceAccountKey.tsx

```diff
@@ -186,33 +188,37 @@ const FormControlServiceAccountKey = ({
   };
 
   const formControl = (
-    <FormControl marginTop="spacingL" id="accountCredentialsFile">
-      <FormControl.Label>Google Service Account Key File</FormControl.Label>
+    <FormControl
+      marginTop="spacingL"
+      id="accountCredentialsFile"
+      isInvalid={formControlState.isInvalid}
+      isRequired={isRequired}
+    >
+      <FormControl.Label isRequired={isRequired}>Google Service Account Key File</FormControl.Label>
```

Comment:
Adding `isRequired` to the parent `FormControl` is enough. It will pass the prop down to its children

Reply:
Whoops, thanks for catching that 👍